test/object-store/audit.cpp

```diff
@@ -1453,7 +1453,7 @@ TEST_CASE("audit management", "[sync][pbs][audit]") {
         audit->wait_for_completion();
 
         auto events = get_audit_events(test_session);
-        REQUIRE(events.size() == 6);
+        REQUIRE(events.size() == 8);
```

Comment:
This is the reason behind the failure, we hare 2 audit events more than expected. Since the events are coming straight from  the table, I guess we have some other issue, we are probably duplicating some data?
```

{"type":"object","value":[{"_id":1,"value":2}]}

{"type":"object","value":[{"_id":1,"value":1}]}

{"type":"object","value":[{"_id":1,"value":0}]}

{"type":"object","value":[{"_id":1,"value":2}]}

{"type":"object","value":[{"_id":1,"value":1}]}

/// this 2 are the same object and they look duplicated..
{"type":"object","value":[{"_id":1,"value":2}]}

{"type":"object","value":[{"_id":1,"value":2}]}

{"type":"object","value":[{"_id":1,"value":2}]}
```

Reply:
oh, actually it is probably due to this: 
`Array m_interner_data;                     // 14th slot in m_top`
Are we are printing the compressed strings? Are we exposing the compressor via the iterable table interface somehow?